opencontrail/compute.sls

```diff
@@ -138,21 +153,32 @@ contrail_load_vrouter_kernel_module:
     - pkg: opencontrail_compute_packages
 
 {%- endif %}
+{%- endif %}
 
 {%- endif %}
 
+{%- if not grains.get('noservices','false')%}
+
 opencontrail_compute_services:
   service.enabled:
   - names: {{ compute.services }}
 
+{%- endif %}
+
 {%- if compute.get('engine', 'openstack') == 'kubernetes' %}
 
 kubernetes_packages:
   pkg.installed:
     - names:
       - bridge-utils
       - ethtool
-      - opencontrail-kubelet
+      - python-pip
+
+opencontrail-kubelet_package:
```

Comment:
No, default installation method is pkg. You could introduce possibility to override these settings using source.engine as for docker compose and other services.